module/documents/active-effect.mjs

```diff
@@ -285,11 +285,12 @@ export default class ActiveEffect5e extends ActiveEffect {
    * @protected
    */
   _prepareExhaustionLevel() {
+    const config = CONFIG.DND5E.conditionTypes.exhaustion;
     let level = this.getFlag("dnd5e", "exhaustionLevel");
     if ( !Number.isFinite(level) ) level = 1;
-    this.icon = `systems/dnd5e/icons/svg/statuses/exhaustion-${level}.svg`;
+    this.icon = `${config.icon.split(".")[0]}-${level}.svg`;
```

Comment:
Since we are going this logic twice, I'd say split it out into a `_getExhaustionIcon` helper method. It would also be a good idea to be cognizant of the extension passed in in case someone wasn't using an SVG or the path includes other dots.

```javascript
  /**
   * Get the image used to represent exhaustion at this level.
   * @param {number} level
   * @returns {string}
   */
  _getExhaustionImage(level) {
    const split = CONFIG.DND5E.conditionTypes.exhaustion.icon.split(".");
    const ext = split.pop();
    const path = split.join(".");
    return `${path}-${level}.${ext}`;
  }